### DISCUSSION TITLE
Is there a way to specify AIDA-X models json and wav via command line, or API?

### DISCUSSION THREAD

**Discussion by cbake6807** [AidaDSP/aida-x#46] [Q&A]
RaspberryPi 4B+ -> Qjackctl -> AIDA-X (post-start script)  live routing for jamming at practice, etc.

![image](https://github.com/user-attachments/assets/fc088564-2f10-4ad0-9205-ba6e7eb80513)

I manged to get it to auto start in proper order, and load my patchbay xml on boot, but of course the default models are loaded. 

Is there any way available to specify an alternate model at all? I'm ideally looking for programmatic methods to set up a way to swap them in and out, eventually via gpio buttons detected on the Pi, and I can at least reload the audio stack every time a button is pushed. 

I have to admit, I was skeptical on the Pi being able to hack this for live use.. Boy, am I impressed! :)

**Comment by falkTX** [comment]
there is an lv2 plugin, and vst2 and vst3 and clap. so you can use any plugin host that supports jack and allows this kind of control.

I believe jalv can do this if you save presets, then you can load them on start.
or carla, that should work too if you save a project and start carla with the project as 1st CLI argument

**Reply by cbake6807** [reply]
Thanks for the quick reply! I will investigate these plugins
